### ISSUE TITLE
Which URL do I have to enter in the Browser for "my-gallery/flush ..."

### ISSUE THREAD

**Issue by ghost** [gidlov/pico-gallery#3]
Hello developer,

sorry that I disturb you again, but I can not realize it.

My gallery page is at http://picocms.wpzweinull.ch/my-gallery/index and I have now added `%my_gallery%` in the index.md.

Which URL must be accurate to generate the thumbnails.

If it is with `mydomain.de/index .....`, then it does not work.

Phile CMS is complicated for me yet a bit.

**Comment by gidlov** [comment]
Hi, I have now discovered that Pico 1.* is not compatible with this plugin.
I will look into this..

Den 5 sep. 2017 16:34 skrev "Alexander Liebrecht" <notifications@github.com

**Comment by ghost** [comment]
Good thank you! I have patience!!

**Comment by gidlov** [comment]
Now its working, download the new version 1.2.2, see readme for info.

**Comment by ghost** [comment]
Hi,

I followed your instructions, but I still do something wrong. On the Gallery Index page, I have the shortcode with the Gallery, but no thumbnails.

This is at http://picocms.wpzweinull.ch/gallery/index

What am I doing wrong?

Thanks in advance.

**Comment by gidlov** [comment]
Have you set page to gallery/index in your config?

2017-09-07 10:38 GMT+02:00 Alexander Liebrecht <notifications@github.com>:

**Comment by ghost** [comment]
Yes, I have now made and the shortcode with "my_gallery" is now no longer visible.

But the thumbnails are not displayed and no thumbnails were generated in the FTP account under `/gallery/thumbnails`.

CHMOD 777 I have for directories like `/gallery` and `/gallery/thumbnails`

In the config.php I made from 'index': 'gallery/index'. As far as everything right I think times, but something is still.

**Comment by gidlov** [comment]
One step closer then. You should enter
http://picocms.wpzweinull.ch/gallery/index/my_gallery/flush/47b71a856274ac45

The gallery name must be the same as the shortcode, for example
%my_gallery% or whatever you named it.

image_path orgins from the pico root, same for thumb_path.






2017-09-07 11:10 GMT+02:00 Alexander Liebrecht <notifications@github.com>:

**Comment by armansansd** [comment]
Hi @gidlov ! 
I installed your plugin on my pico, and everything work fine except that no thumbnail are create in my thumbnail folder. Despite the fact that I chmod my folder to 777 (like @AlexL777 ) 
Proper links are generated on my index page. So i guess the problem come from the thumbnails generation with intervention. But it seems that it is properly install with composer, is there a way to debug the plugin so I can find from where the error come from ? Or maybe you've through this problem already. Could it come from the htaccess ? 

I'm using pico 1.0.6, I looking for clues :)

thx a lot

**Comment by gidlov** [comment]
Does the flush config value match the flush URL? Does the %gallery_name%
match the value in gallery config? Does the %gallery_name% dissapear?

Den fre 4 maj 2018 kl 23:41 skrev Arman Mohtadji <notifications@github.com>:

**Comment by armansansd** [comment]
Hi ! 
I started all over on a fresh install, 
and there is some evolutions, 
1. the flush creat the thumbnail in the right place but only if I add 
` content/gallery/fullsize` to the config file line. Instead of `gallery/fullsize`.
is-it normal ? 
2. The url displayed in the img tag is :     
```html
<img src="http://localhost/robots/content/gallery/thumbnails/thumb_image.png" alt="" class="">
```
Which of course is not  to any images. 


To responde to your previous questions, 
my gallery folder is called ```gallery```, my gallery array is called ```my_gallery``` , and I paste ```%my_gallery%``` in the index.md file. 
and yes the %my_gallery% disappear and is replace by a 
```html
<p>
<a href="http://localhost/robots/index/my_gallery/image.png">
<img src="http://localhost/robots/content/gallery/thumbnails/thumb_image.png">
</a>
</p>
```   
Which is weird cause i got ```my_gallery``` then ```gallery```.


I'm so close ! I'm sure is due to my lack of knowledge in pico's behavior. 

Thank you for your patience 

Cheers

**Comment by gidlov** [comment]
That's right, it goes from /content /*. If you update the Pico and Pico
Gallery to latest versions starts from the /*.

Den lör 5 maj 2018 kl 22:37 skrev Arman Mohtadji <notifications@github.com>:

**Comment by malu58** [comment]
Hello Klas,

it's possible to imlement your gallery plugin on  nextcloud 13.0.5 - PicoCMS 0.9.7 app?
